actionview/lib/action_view/helpers/tags/password_field.rb

```diff
@@ -3,7 +3,7 @@ module Helpers
     module Tags # :nodoc:
       class PasswordField < TextField # :nodoc:
         def render
-          @options = {:value => nil}.merge!(@options)
+          @options = {:value => nil, :autocomplete => 'off' }.merge!(@options)
```

Comment:
:smile:  exactly same thing my collegue told me few hours ago :) , ...yes highly agree ,I was just following style the original code was written. 

I'll change this if more people are ok with this, ...but it seems I may need to change the entire  pull request as the discussion on the issue is shifting against my proposal, and I'm starting to see why :)